packages/pigeon/tool/shared/test_runner.dart

```diff
@@ -18,31 +18,67 @@ Future<void> runTests(
   bool runFormat = false,
   bool runGeneration = true,
   bool ciMode = false,
+  bool overflow = false,
```

Comment:
I would call this one `inculdeOverflow`. The other instances of this name all seem to be a switch controlling which of the two modes the method is in, but here the modes are "without" and "without and with".